pcbdevice/opencrcontroler/GcodeInterpreter.ino

```diff
@@ -172,6 +239,31 @@ if (Serial.available())
 // High level Functions :
 //########################
 
+int Homing(uint8_t IDX, uint8_t IDY, uint8_t IDZ){
+  homing = 1;
+  // home Z :
+  homingZ = HIGH;
+  homingY = LOW;
+  homingX = LOW;
+  Torque_on(IDZ);
+  Wheel(IDZ, -50);
+  
+  // home y :
+  homingZ = LOW;
+  homingY = HIGH;
+  homingX = LOW;
+  Torque_on(IDY);
+  Wheel(IDZ, -100);
+  
+  // home x :
+  homingZ = LOW;
+  homingY = LOW;
+  homingX = HIGH;
+  Torque_on(IDX);
+  Wheel(IDZ, -100);
+  
+}
+
 int MovingDistance(String axe, float value_mm, uint8_t IDX, uint8_t IDY, uint8_t IDZ){

```

Comment:
Remove this function
Create a function to convert X/Y/Z to idX/Y/Z
Create a function to convert mm to tick

Reply:
done